app/src/main/res/values-ja/localized.xml

```diff
@@ -1,2 +1,424 @@
+    <string name="p_fine_tune_support_max_updates">更新回数上限</string>
+    <string name="p_fine_tune_similarity">類似度</string>
+    <string name="p_fine_tune_min_similarity">最小の類似度</string>
+    <string name="p_fine_tune_mlb_similarity">MLBの類似度</string>
+    <string name="p_fine_tune_stage_counter_similarity">ウェーブカウンターの類似度</string>
+    <string name="p_fine_tune_clicks">クリック</string>
+    <string name="p_fine_tune_wait_after_clicking">クリック後の待機秒数</string>
+    <string name="p_fine_tune_click_duration">クリックの継続時間</string>
+    <string name="p_fine_tune_click_delay">マルチクリック間隔</string>
+    <string name="p_fine_tune_swipes">スワイプ</string>
+    <string name="p_fine_tune_wait_after_swiping">スワイプ後の待機秒数</string>
+    <string name="p_fine_tune_swipe_duration">スワイプの継続時間</string>
+    <string name="p_fine_tune_swipe_multiplier">スワイプ時間倍率</string>
+    <string name="p_fine_tune_wait">待機</string>
+    <string name="p_fine_tune_skill_delay">スキル使用後の待機時間</string>
+    <string name="p_fine_tune_wait_before_turn">ターン経過後の待機時間</string>
+    <string name="p_fine_tune_wait_before_cards">カード選択後の待機時間</string>
+    <string name="p_fine_tune_wait_multiplier">待機時間倍率</string>
+
+    <string name="troubleshoot">Troubleshooting</string>
+    <string name="battle_config_edit">バトルの設定</string>
+    <string name="card_priority">カードの優先順位</string>
+
+    <string name="p_refill_type_gold">黄金の果実</string>
+    <string name="p_refill_type_silver">白銀の果実</string>
+    <string name="p_refill_type_bronze">青銅の果実</string>
+    <string name="p_refill_type_copper">赤銅の果実</string>
+    <string name="p_refill_type_sq">聖晶石</string>
+
+    <string name="p_brave_chains_don_t_care">考慮しない</string>
+    <string name="p_brave_chains_with_np">宝具と同時に</string>
+    <string name="p_brave_chains_avoid">使用しない</string>
+
+    <string name="p_boost_item_disabled">使用確認OFF</string>
+    <string name="p_boost_item_skip">使用しない</string>
+
+    <string name="spam">リピートモード</string>
+    <string name="spam_target">ターゲット</string>
+    <string name="spam_np">宝具:</string>
+    <string name="p_spam_none">なし</string>
+    <string name="p_spam_spam">可能なときに使用</string>
+    <string name="p_spam_danger">DANGER</string>
+
+    <string name="p_support_mode_first">一番上を選択</string>
+    <string name="p_support_mode_manual">手動</string>
+    <string name="p_support_mode_preferred">優先</string>
+    <string name="p_support_mode_friend">フレンド選択</string>
+
+    <string name="p_script_mode_battle">バトル</string>
+    <string name="p_script_mode_lottery">箱イベ</string>
+    <string name="p_script_mode_fp">フレンドポイント召喚</string>
+    <string name="p_script_mode_support_image_maker">サポートイメージメーカー</string>
+    <string name="p_script_mode_gift_box">プレゼントボックス</string>
+
+    <string name="p_max_gold_ember_set_size">次の個数以下の種火を選択する</string>
+    <string name="p_max_gold_ember_total_count">受け取った種火の総数</string>
+
+    <string name="p_not_set">未設定</string>
+
+    <string name="p_support_class_none">なし</string>
+    <string name="p_support_class_all">All</string>
```

Comment:
It is not directly translated, but it is considered easy to understand the indications in line with the game notation.

Reply:
I'm asking because you translated mat_monument_saber, mat_piece_saber, mat_skill_gold_saber and so on, but these are the only ones you didn't translate with `セイバー`, `アーチャー`, ...

Reply:
This is because English is used on the FGO screen in situations where this notation is used.